docs/cve-creation-guide.md

```diff
@@ -0,0 +1,189 @@
+# CVE Creation Guide
+This is a guide to creating a CVE request via Mitre.
+It may be of some use to explain what the CVE fields are looking for with some hints and examples.
+
+# Creating a CVE Request
+
+This is an example of how to create a CVE with an older vulnerability from Mojolicious that did not have one previously assigned.
+
+  * CPANSA-Mojolicious-2018-01
```

Comment:
This might muddy the waters a bit, I don't think CPANSA identifiers are relevant for CVEs (except as URL references, but there are currently no way of linking to CPANSA resources)

Reply:
True but it is the specific issue in this example